<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>DwarfClipboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DwarfClipboard.qrc">
    <normaloff>:/icons/images/DwarfClipboard.png</normaloff>:/icons/images/DwarfClipboard.png</iconset>
  </property>
  <widget class="QTabWidget" name="TabWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabRecent">
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Recent</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="DwarfClipboardTableView" name="tableViewRecent">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonRecentLoad">
         <property name="toolTip">
          <string>Load selected layout to file</string>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRecentSave">
         <property name="toolTip">
          <string>Save selected layout to file</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonToLibrary">
         <property name="toolTip">
          <string>Add selected layout to file</string>
         </property>
         <property name="text">
          <string>Add to Library</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRecentSetCursorPrev">
         <property name="toolTip">
          <string>Sets the cursor to the start of the last paste location</string>
         </property>
         <property name="text">
          <string>Set Cursor To
Previous Paste</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRecentPasteDesignations">
         <property name="toolTip">
          <string>Paste selected layout designations to DF</string>
         </property>
         <property name="text">
          <string>Paste
Designations</string>
         </property>
         <property name="shortcut">
          <string comment="Ctrl+SHIFT+D"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRecentPasteBuildings">
         <property name="toolTip">
          <string>Paste selected layout buildings to DF</string>
         </property>
         <property name="text">
          <string>Paste
Buildings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabLibrary">
    <attribute name="title">
     <string>Library</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="DwarfClipboardTreeView" name="treeViewLibrary">
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonLibraryLoad">
         <property name="toolTip">
          <string>Load selected layout from file</string>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibrarySaveSelected">
         <property name="toolTip">
          <string>Save selected layout to file</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibraryReload">
         <property name="toolTip">
          <string>Reload library from disk</string>
         </property>
         <property name="text">
          <string>Reload Library</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibrarySave">
         <property name="toolTip">
          <string>Save library to disk</string>
         </property>
         <property name="text">
          <string>Save Library</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibrarySetCursorPrev">
         <property name="toolTip">
          <string>Sets the cursor to the start of the last paste location</string>
         </property>
         <property name="text">
          <string>Set Cursor To
Previous Paste</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibraryPasteDesignations">
         <property name="toolTip">
          <string>Paste selected layout designations to DF</string>
         </property>
         <property name="text">
          <string>Paste
Designations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLibraryPasteBuildings">
         <property name="toolTip">
          <string>Paste selected layout buildings to DF</string>
         </property>
         <property name="text">
          <string>Paste
Buildings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabOptions">
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="copyShortcutLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copy Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="copyShortcutButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set shortcut to copy from DF</string>
       </property>
       <property name="text">
        <string>Ctrl-Alt-C</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pasteDesignationShortcutLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Paste Designation Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pasteDesignationShortcutButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set  shortcut to paste designations to DF</string>
       </property>
       <property name="text">
        <string>Ctrl-Alt-D</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="thumbnailSizeLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thumbnail Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="thumbnailSizeLineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set thumbnail size</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="inputDelayMsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Input Delay (ms):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputDelayMsLineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set input delay, higher is longer</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="tilesetPathLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tileset Path:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="tilesetPathButton">
         <property name="toolTip">
          <string>Select current tileset</string>
         </property>
         <property name="text">
          <string>C:/DF/16x16.png</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tilesetPathToolButton">
         <property name="toolTip">
          <string>Read current tileset from DF</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DwarfClipboard.qrc">
           <normaloff>:/icons/images/cog.png</normaloff>:/icons/images/cog.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="colorsPathLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Colors Path:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="colorPathButton">
         <property name="toolTip">
          <string>Select color file</string>
         </property>
         <property name="text">
          <string>C:/DF/colors.png</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="colorPathToolButton">
         <property name="toolTip">
          <string>Read current color from DF</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="DwarfClipboard.qrc">
           <normaloff>:/icons/images/cog.png</normaloff>:/icons/images/cog.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="useOriginalTilesetImagesLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Use Original Tileset Images:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="useOriginalTilesetImagesCheckBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>check to use the original tileset when copied, uncheck to use the current tileset</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="setCursorToPreviousPasteShortcutLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Set Cursor to Previous Paste Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="setCursorToPreviousPasteShortcutButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set shortcut to move the cursor to the start of the last paste location</string>
       </property>
       <property name="text">
        <string>Ctrl-Alt-P</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pasteBuildingShortcutLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Paste Building Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pasteBuildingShortcutButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set  shortcut to paste buildings to DF</string>
       </property>
       <property name="text">
        <string>Ctrl-Alt-B</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DwarfClipboardTableView</class>
   <extends>QTableView</extends>
   <header>DwarfClipboardtableview.h</header>
  </customwidget>
  <customwidget>
   <class>DwarfClipboardTreeView</class>
   <extends>QTreeView</extends>
   <header>DwarfClipboardtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DwarfClipboard.qrc"/>
 </resources>
 <connections/>
</ui>
